iphone/TitaniumKit/TitaniumKit/Sources/API/TiExceptionHandler.m

```diff
@@ -156,8 +156,16 @@ - (id)initWithDictionary:(NSDictionary *)dictionary
     if (_backtrace == nil) {
       _backtrace = [[[dictionary objectForKey:@"stack"] description] copy];
     }
-    _nativeStack = [[dictionary objectForKey:@"nativeStack"] copy];
     _dictionaryValue = [dictionary copy];
+
+    id nativeStackObject = [dictionary objectForKey:@"nativeStack"];
+    if ([nativeStackObject isKindOfClass:[NSArray class]]) {
```

Comment:
We should validate if `nativeStackObject` is `nil` since `isKindOfClass` would fail if it is